### DISCUSSION TITLE
Flagged Acount

### DISCUSSION THREAD

**Discussion by LouaiKhodary13** [community/community#148082] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

my friend tried to add me to a repo 
but its says user cant be added

**Comment by sambitsargam** [comment]
It sounds like the GitHub account might be flagged, which is preventing your friend from adding you to a repository. Check your GitHub notifications or emails for any messages about your account status. If flagged, follow the instructions from GitHub to resolve the issue or contact GitHub support for further assistance. Ensure also that the repository settings and your friend's permissions allow adding collaborators.

**Reply by LouaiKhodary13** [reply]
nothing is in my email or notification regarding the flagged account 
I opend Github in a scanted counfty and when I came back to Egypt I cant be added to new repos

**Comment by mecodeatlas** [comment]
Hi @LouaiKhodary13! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
